pubtools/_quay/container_image_pusher.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, push_items, target_settings):
 
         self.quay_host = self.target_settings.get("quay_host", "quay.io").rstrip("/")
         self._quay_client = None
+        self._quay_client_osbs = None
```

Comment:
I would prefer if the names were something like "source_quay_client" and "dest_quay_client". It would be more self-explanatory that way

Reply:
Updated.